scripts/qa_crowbarsetup.sh

```diff
@@ -3334,6 +3335,7 @@ function deploy_single_proposal()
             ;;
         trove)
             iscloudver 5plus || return
+            [[ $want_trove = 1 ]] || return
```

Comment:
This is correct, this is equal to
```
if [[ $want_trove != 1 ]]; then
  return
fi
```
We don't want to apply trove proposal when option want_trove is different than 1